ci/scripts/audit.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -eux
+
+pushd dp-dataset-exporter-xlsx
+    mvn ossindex:audit
```

Comment:
Should this be calling `make audit`? This is how it was done on babbage although there it was to make it easier to do both java and js auditing from the same step so might not be a consideration here.

Reply:
Good spot 😴